src/views/DocumentCompare.vue

```diff
@@ -123,7 +124,8 @@ export default {
 					toA: matches.toA,
 					fromB: matches.fromB,
 					toB: matches.toB,
-					color: colorForIndex(index)
+					color: colorForIndex(index),
+					index: index
```

Comment:
if key name and value is same, there's no need to do `index: index`, just `index`: is sufficient.